test/src/org/labkey/test/tests/targetedms/TargetedMSAuditLogTest.java

```diff
@@ -57,4 +51,40 @@ public void testAuditLogImported()
                 auditLog.getDataAsText(8, "MessageText"));
 
     }
+
+    /*
+        Test coverage for Issue 44507: Importing the same Skyline document in multiple containers associates the audit log solely with the last import
+     */
+    @Test
+    public void testAuditLogInTwoDifferentFolders()
+    {
+        log("Creating folder and importing same SKY file");
+        goToProjectHome();
+        String FOLDER_2 = "Folder 2 with same audit logs";
+        setUpFolder(FOLDER_2, FolderType.QC);
+        importData(AuditTrail_FILE);
+
+        log("Verifying logs are imported correctly in " + FOLDER_2);
+        DataRegionTable auditLog = getAuditLogs(FOLDER_2);
+        assertEquals("Invalid number of audit logs", 9, auditLog.getDataRowCount());
```

Comment:
We need to verify that we can see the audit data in *both* folders when both copies are in the DB. This is part of step 2 in my test description in the issue.

Reply:
Done